datafusion/sqllogictest/test_files/strings.slt

```diff
@@ -115,6 +115,12 @@ p1
 p1e1
 p1m1e1
 
+query T rowsort
```

Comment:
In case anyone is curious, without the fix this test fails like this:

```
Completed 1 test files in 0 seconds                                                                                                                                                                                             External error: query result mismatch:
[SQL] SELECT s FROM test WHERE s ILIKE 'p1';
[Diff] (-expected|+actual)
-   P1
    p1
at test_files/strings.slt:118
```